NuRadioReco/modules/io/coreas/coreas.py

```diff
@@ -113,3 +120,34 @@ def make_sim_station(station_id, corsika, observer, channel_ids,  weight=None):
     sim_station.set_is_cosmic_ray()
     sim_station.set_simulation_weight(weight)
     return sim_station
+
+
+def make_sim_shower(corsika):
+    sim_shower = NuRadioReco.framework.radio_shower.RadioShower()
+
+    zenith, azimuth, magnetic_field_vector = get_angles(corsika)
+    sim_shower.set_parameter(shp.zenith, zenith)
+    sim_shower.set_parameter(shp.azimuth, azimuth)
+    sim_shower.set_parameter(shp.magnetic_field_vector, magnetic_field_vector)
+
+    energy = corsika['inputs'].attrs["ERANGE"][0] * units.GeV
+    sim_shower.set_parameter(shp.energy, energy)
+    sim_shower.set_parameter(shp.core, np.array([0, 0, 0]))
+
+    sim_shower.set_parameter(shp.shower_maximum, corsika['CoREAS'].attrs['DepthOfShowerMaximum'] * units.g / units.cm2)
+    sim_shower.set_parameter(shp.refractive_index_at_ground, corsika['CoREAS'].attrs["GroundLevelRefractiveIndex"])
+    sim_shower.set_parameter(shp.magnetic_field_rotation,
+                             corsika['CoREAS'].attrs["RotationAngleForMagfieldDeclination"] * units.degree)
+    sim_shower.set_parameter(shp.distance_shower_maximum_geometric,
+                             corsika['CoREAS'].attrs["DistanceOfShowerMaximum"] * units.cm)
+
+    sim_shower.set_parameter(shp.observation_level, corsika["inputs"].attrs["OBSLEV"] * units.cm)
+    sim_shower.set_parameter(shp.primary_particle, corsika["inputs"].attrs["PRMPAR"])
+    sim_shower.set_parameter(shp.atmospheric_model, corsika["inputs"].attrs["ATMOD"])
```

Comment:
This property doesn't exist in out CoREAS files. Could you check for that before adding it?

Reply:
In newly converted hdf5 files it should exists (I added it to the hdf5 converter in the CORSIKA/CoREAS trunk, and it should also be in the newest release). So for the moment it might be practical to check if it is in but i think for the long run we want this to be there in any case I think.

Reply:
Well, our files are a bit older. There is no harm in letting people read older files